datalogtool/publish.gradle

```diff
@@ -34,7 +34,7 @@ model {
                             description("Copies the DataLogTool executable to the outputs directory.")
                             destinationDirectory = outputsFolder
 
-                            archiveBaseName = '_M_' + zipBaseName
+                            archiveBaseName = zipBaseName
```

Comment:
Could we just use zipBaseName for this instead now?

Reply:
What do you mean? archiveBaseName is the gradle property that is being set.

Reply:
Ohh ok I didn't realise this was setting a property 🤦‍♀️